### ISSUE TITLE
[Feature request] Tooltip

### ISSUE THREAD

**Issue by liubko** [swiftcarrot/react-input-slider#48]
Would be awesome if component supports tooltip out of the box
<img width="185" alt="Screenshot 2019-11-18 at 18 40 05" src="https://user-images.githubusercontent.com/4257937/69076013-db2ca480-0a32-11ea-98a9-c9c448aab67c.png">

**Comment by wangzuo** [comment]
Yes, we are working on a release with marker and tooltip on slider

**Comment by jaydeesigns** [comment]
@wangzuo This would be an excellent addition. By **marker** do you mean the ability to display min/max on both ends?

**Comment by juliatres** [comment]
Will it be available for xy too?
